### PR TITLE
Added message in v5 cluster view when there are no node pools

### PR DESCRIPTION
**Pull request by ograu**
Captura de pantalla 2019-10-16 a las 13 07 20

### CODE REVIEW

File: src/components/cluster/detail/cluster_detail_node_pools_table.js [hunk 1]
```diff
@@ -222,6 +222,18 @@ const FlexWrapperDiv = styled.div`
   button {
     margin-right: 16px;
   }
+  &.no-nodepools {
+    flex-direction: column;
+    justify-content: space-between;
+    height: 147px;
```
Comment (axbarsan) [inline src/components/cluster/detail/cluster_detail_node_pools_table.js]: This is really specific haha

Comment (ograu) [reply src/components/cluster/detail/cluster_detail_node_pools_table.js]: You mean the height? Yeah, this is what I found in wireframes, and BTW I forgot to add an image for this: 

Captura de pantalla 2019-10-16 a las 13 07 20

File: src/components/cluster/detail/cluster_detail_node_pools_table.js [hunk 2]
```diff
@@ -561,7 +574,13 @@ class ClusterDetailNodePoolsTable extends React.Component {
             </AddNodePoolWrapperDiv>
           </ReactCSSTransitionGroup>
         ) : (
-          <FlexWrapperDiv>
+          <FlexWrapperDiv className={noNodePools && 'no-nodepools'}>
+            {noNodePools && (
+              <p>
+                Add at least one node pools to this cluster so that you can
```
Comment (axbarsan) [inline src/components/cluster/detail/cluster_detail_node_pools_table.js]: I think this should say "add at least one node pool", (without the "s" at the end)

Comment (ograu) [reply src/components/cluster/detail/cluster_detail_node_pools_table.js]: good catch, thanks!

**Pull request merged**